README.md

```diff
@@ -1,3 +1,11 @@
 # minesweeper-swift
 
 Minesweeper - written in Swift and SpriteKit, streamed live on Twitch
+
+## Videos
+
+- Twitch
+  - https://www.twitch.tv/videos/592820582
```

Comment:
This twitch recording is ephemeral, so I think the Youtube Link is sufficient.

Reply:
I did second guess it and nearly put a comment about it going, I’ll remove it